portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -31,34 +31,37 @@
 
 /** Servlet for handling comments data */
 @WebServlet("/comments")
-public class DataServlet extends HttpServlet {
+public class CommentServlet extends HttpServlet {
 
-  /* Datastore data is represented by entities which have a kind and certain properties,
-   * and the constants below define the kind and property names for comment entities.
-   * Changing them would mean comments saved with a previous definition would not be displayed anymore. */
-  private static final String COMMENT_KIND = "Comment";
-  private static final String CONTENT_PROPERTY = "content";
-
-  /* This is the name of the input form which is defined in the html of the main page */
+  /* The name of the input form which is defined in the html of the main page. */
```

Comment:
Comments are generally good - but code which doesn't need comments is better :) I think in this case you could just rename the constant to INPUT_FORM_NAME and remove the comment.